newdle/client/src/components/login/LoggingIn.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import {Trans} from '@lingui/macro';
 import {Dimmer, Loader} from 'semantic-ui-react';
 
 export default function LoggingIn() {
   return (
     <Dimmer active page>
-      <Loader size="massive">Logging in...</Loader>
+      <Loader size="massive"><Trans>Logging in...</Trans></Loader>
```

Comment:
Prettier wants to reformat this:

```suggestion
      <Loader size="massive">
        <Trans>Logging in...</Trans>
      </Loader>
```

Reply:
ok sure